src/clean.cc

```diff
@@ -112,7 +112,7 @@ void Cleaner::PrintFooter() {
 int Cleaner::CleanAll(bool generator) {
   Reset();
   PrintHeader();
-  for (vector<Edge*>::iterator e = state_->edges_.begin();
+  for (vector<Edge*>::const_iterator e = state_->edges_.begin();
```

Comment:
Same as above.

Reply:
Note to myself: Nope, `cbegin` is C++11.